### ISSUE TITLE
Event Duration Statistical error

### ISSUE THREAD

**Issue by MJHou** [getsentry/sentry-java#2991]
### Integration

sentry-android

### Build System

Gradle

### AGP Version

6.5

### Proguard

Enabled

### Version

6.25.0

### Steps to Reproduce

Hello, after upgrading Sentry to 6.25.0, the Event Duration that requires the page has been particularly impressive for nearly a minute. I specifically searched for a sample and found that the starting statistical time for events is from the start of application startup. Here is a screenshot:
<img width="414" alt="image" src="https://github.com/getsentry/sentry-java/assets/14875242/8959bbc1-6ac8-4423-8e72-42906ef81b5f">
<img width="505" alt="image" src="https://github.com/getsentry/sentry-java/assets/14875242/d7e7f53d-0871-46e9-956a-9bd0b0199cbf">
<img width="499" alt="image" src="https://github.com/getsentry/sentry-java/assets/14875242/c650b38b-c9fd-44d2-b21c-afa1a67bae04">
 Do you know what caused this?
 

### Expected Result

Event Duration  140ms

### Actual Result

This is the log
[json.txt](https://github.com/getsentry/sentry-java/files/12923739/json.txt)

**Comment by MJHou** [comment]
Hello, have you reached a conclusion on this question?

**Comment by kahest** [comment]
Hey @MJHou thanks for the report, we will look into this.

**Comment by MJHou** [comment]
@kahest Do you have corresponding documentation explaining the statistical rules of your data? I will also investigate if there is a connection issue on my end.

**Comment by romtsn** [comment]
@MJHou can you elaborate a bit on what you mean by "statistical"?

**Comment by MJHou** [comment]
How is the value of Event Duration calculated? By the way, we have added network monitoring.

**Comment by markushi** [comment]
Hey @MJHou,
Thanks for providing the json file!

Taking a closer look at it I can see the following span, where `timestamp - start_timestamp` results into the 42 seconds.
```json
      "timestamp": 1697433826.329404,
      "start_timestamp": 1697433783.771,
      "description": "OneClickLoginDialogActivity initial display",
      "op": "ui.load.initial_display",
```

Could you explain your app start behaviour a bit more? Is `OneClickLoginDialogActivity` a dialog or an Activity?

**Comment by markushi** [comment]
Alright I dug a bit deeper and it looks like this is a bug on our end when either appStart/coldStart could not be determined.
https://github.com/getsentry/sentry-java/blob/02e9e80a782baadbfb6211aa83f3bedcd1df6b42/sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java#L221-L228

@MJHou could you give us a bit more details on how/when you're initializing the sentry SDK?

**Comment by MJHou** [comment]
OneClickLoginDialogActivity is an Activity。It's just a problem with naming。
Can you explain your application startup behavior again? What does this mean?
The user simply clicks on the application icon on the desktop. Since they go to the Splash page, the declaration cycle of each activity can be seen in the JSON file

**Comment by MJHou** [comment]
OneClickLoginDialogActivity is an Activity。It's just a problem with naming。
Can you explain your application startup behavior again? What does this mean?
The user simply clicks on the application icon on the desktop. Since they go to the Splash page, the declaration cycle of each activity can be seen in the JSON file

**Comment by MJHou** [comment]
SentryAndroid.init(context, options -> {
    options.setDebug(BuildConfig.DEBUG);
    options.setDsn(Config.SENTRY_URL);
    options.setTracesSampleRate(1.0);
    options.setEnableSystemEventBreadcrumbs(false); 
});
This is our initialization code. We initialized it in onCreat() of Application

**Comment by MJHou** [comment]
Hello, do you have any conclusions about this issue now?

**Comment by markushi** [comment]
@MJHou Yes, we identified an issue on our end. Good news: This is likely fixed with https://github.com/getsentry/sentry-java/pull/3092, once a release is out it would be great if you could give it a try.

**Comment by markushi** [comment]
Fixed with the `7.1.0` release, if you're still seeing an issue feel free to re-open this ticket.
